lib/core/src/server/config/utils.js

```diff
@@ -23,6 +23,7 @@ export function loadEnv(options = {}) {
     // even when deployed inside a subpath. (like in GitHub pages)
     // In development this is just empty as we always serves from the root.
     PUBLIC_URL: options.production ? '.' : '',
+    PREVIEW_URL: process.env.PREVIEW_URL,
```

Comment:
maybe this should be `|| ''` as in the other options above

Reply:
will do

Reply:
pushed